spdx/writers/rdf.py

```diff
@@ -955,9 +952,8 @@ def pkg_ext_refs(self):
         """
         Return a list of package external references.
         """
-        return map(
-            self.create_package_external_ref_node, self.document.package.pkg_ext_refs
-        )
+        return [self.create_package_external_ref_node(ext_ref) for package in self.document.packages
+                for ext_ref in package.pkg_ext_refs]
```

Comment:
Normally, external references are part of a package and should not be gathered all in one place, but the rdf writer seems to handle it that way, so we leave that for later.

Reply:
I agree and this is only some kind of workaround to not break the functionality but this will definitely need more work on the rdf writer..